sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

```diff
@@ -1,3093 +1,3258 @@
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "name": "devices",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/ExportImportDevice"
-              }
-            }
-          },
-          {
-            "$ref": "#/parameters/api-version"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/BulkRegistryOperationResult"
-            }
-          },
-          "400": {
-            "description": "BadRequest",
-            "schema": {
-              "$ref": "#/definitions/BulkRegistryOperationResult"
-            }
-          }
-        }
-      }
-    },
-    "/devices/query": {
-      "post": {
-        "summary": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language.",
-        "description": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language for more information. Pagination of results is supported. This returns information about device twins only. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-        "operationId": "RegistryManager_QueryIotHub",
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "name": "querySpecification",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/QuerySpecification"
-            }
-          },
-          {
-            "$ref": "#/parameters/api-version"
-          },
-          {
-            "name": "x-ms-continuation",
-            "in": "header",
-            "required": false,
-            "type": "string"
-          },
-          {
-            "name": "x-ms-max-item-count",
-            "in": "header",
-            "required": false,
-            "type": "string"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Query result with continuation token if appropriate.",
-            "schema": {
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/Twin"
-              }
+    "swagger": "2.0",
```

Comment:
I'm not sure why this shows the diff this way but it does not look specific to the messaging APIs. Also we should only be modifying the other iothubservice_modified json file only.

Reply:
I agree. I have modified `iothubservice.json`, but only to highlight the difference between the supplied swagger and the proposed changes. I can definitely move it to `iothubservice_modified.json`. 
That being said, this PR is only to discuss the recommendation before we give them to the service team.

This PR is not intended to be merged into `master`.